test/experimental/eventtype_autocreate_test.go

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 package experimental
 
+//go:generate go.work
```

Comment:
Is it related to this containersource test? Is it necessary? It looks unrelated.

Reply:
I have removed that. Thanks for pointing that out.